src/main/java/seedu/edulink/logic/Messages.java

```diff
@@ -57,4 +58,16 @@ public static String format(Student student) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code grade} for display to the user.
+     */
+    public static String format(Grade grade) {
```

Comment:
Is it best to use this function ? Should we not keep to the standard of showing the student we updated / added the grade with. I believe you can modify the format::Student function.